deployment/k8s-config/kustomize/base/skaha/config/launch-carta.yaml

```diff
@@ -72,6 +72,8 @@ spec:
           protocol: TCP
           name: frontend-port
         volumeMounts:
+        - mountPath: "/skaha-system"
+          name: start-desktop
```

Comment:
typo... 'start-carta'... and in the volumes below

Reply:
I just kept it consistent with the other launch scripts.  I will change.